pkg/tracing/lightstep/lightstep.go

```diff
@@ -58,3 +69,30 @@ func NewTracer(ctx context.Context, yamlConfig []byte) (opentracing.Tracer, io.C
 	}
 	return t, t, nil
 }
+
+// parseTags parses the given string into a map of strings to empty interface.
+// Spec for this value:
+// - comma separated list of key=value
+// - value can be specified using the notation ${envVar:defaultValue}, where `envVar`
+// is an environment variable and `defaultValue` is the value to use in case the env var is not set.
+func parseTags(sTags string) opentracing.Tags {
```

Comment:
Can we have a quick unit-test for it? (:

Reply:
I was thinking about that 😜  But there aren't any other unit tests in the tracing package (except stackdriver), so I figured it didn't make sense to break that ground for this simple change.

Should we add in a testing file and scaffolding for this?

Reply:
Scratch all that... I went ahead and added testing. PTAL